docs/hosting/configuration/environment-variables/endpoints.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Endpoints environment variables
+description: Customize the application's API and webhook endpoints with environment variables for your self-hosted n8n instance. 
+contentType: reference
+tags:
+  - environment variables
+hide:
+  - toc
+  - tags
+---
+
+# Endpoints environment variables
+
+This section outlines environment variable configurations for setting payload size limits, enabling and customizing metrics, managing REST and webhook endpoints, and providing options for proxy compatibility.
```

Comment:
Try this one in https://hemingwayapp.com/ As you'll see it's a very high reading grade. Hemingway suggests aiming for 9 or below. In docs I usually try to go even lower. Play around in Hemingway and see if you can rephrase this.

Reply:
Shortened & simplified :)